docs/landing/layouts/index.html

```diff
@@ -1,34 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    {{ partial "meta.html"}}
-    <link rel="shortcut icon" href="{{.Site.BaseUrl}}/s/img/favicon.png">
+    {{- partial "meta.html" -}}
```

Comment:
Go added whitespace trimming into its template language and now the reference index page size has been reduced from over 3000 lines to 350.